### ISSUE TITLE
Sync add-on settings to Firefox Sync

### ISSUE THREAD

**Issue by cfoellmann** [eric-bixby/auto-sort-bookmarks-webext#13]
It would be great to sync the settings to the firefox sync service

**Comment by eric-bixby** [comment]
This appears to be possible:
https://support.mozilla.org/en-US/questions/1014502

**Comment by cfoellmann** [comment]
I have no idea when it comes to coding for ff.
This should be a "one-liner"!?

**Comment by eric-bixby** [comment]
Nope, more than one line.  Although, shouldn't be too complicated.

**Comment by Gitoffthelawn** [comment]
It will be a nice feature, but please make it optional!

**Comment by eric-bixby** [comment]
Preferences for ASB 3.1 (Web-Extensions API) are stored under storage.local.  I need research using storage.sync instead.

**Comment by eric-bixby** [comment]
Sorry it's been a while but it is still a good feature.
